docs/user-guide/query-data.md

```diff
@@ -6,28 +6,29 @@ GreptimeDB supports full SQL for you to query data from a database. Here are som
 To select all the data from the `system_metrics` table, use the `SELECT` statement:
```

Comment:
```suggestion
To select all the data from the `monitor` table, use the `SELECT` statement:
```

And the sentence above need to be modified too

Reply:
sharp eyes 👍